pkg/framework/guest.go

```diff
@@ -108,7 +120,7 @@ func (g *Guest) Setup() error {
 func (g *Guest) WaitForAPIDown() error {
 	time.Sleep(1 * time.Second)
 
-	log.Println("level", "debug", "message", "waiting for k8s API to be down")
+	g.logger.Log("level", "debug", "message", "waiting for k8s API to be down")
```

Comment:
Does that mean we'll have JSON in circle output?

Reply:
I do not understand the problem. It is configurable. You decide what you do in your code. When you want to do something else you change your code.

Reply:
To change to output, you have to manipulate the code, instead out having a flag like `--micrologger-output=json`.

Second thing is that when you want to change something you need make sure every logger instance in the code is changed, because we always want all the loggers to be aligned. That is a huge chore, and I don't see benefits.

Reply:
When you want to change it everywhere you change the default in micrologger. Come on Pawel. What is the problem?

Reply:
The problem is changing it per project basis. I don't want to change the default as this is good default most cases.

Reply:
This is not a problem. We can just do it if we want.

Reply:
We can. It's unnecessarily difficult.

Reply:
The downsides and side effects we would have with anything globally configured would just be worse. That does not make any sense to me.